### PR TITLE
First journey page & journey step template

### PR DESCRIPTION
**Pull request by virginiacc**
## Additions
- First journey/`know the process` page (`prepare` step)
- `.process-step` template
- `know-the-process.less`

## Removals
- duplicated `cf-typography.less` styles from `typography.less`

## Notes
- The css still needs some tweaks, but I'd like to get this out so we can start discussion
- Since it seems like each of the step pages will have similar content, I've created a shared `_process-step` template. The template can be populated with step-specific data declared in the `index.html` page for each individual step. 
- The journey naming/path conventions are still under discussion, so for now I'm putting the steps in a `process` directory with the first step in the main `index.html`.

## Screenshots
owning a home know the process gt consumer financial protection bureau

### CODE REVIEW

**Comment by stephanieosan** [comment]
@virginiacc Do we know why this is failing? Is it in your power to fix it?

**Comment by fna** [comment]
@stephanieosan @virginiacc here https://travis-ci.org/cfpb/owning-a-home/builds/52067527 for one of the tests it says "use of const in strict mode", I assume one can't use ``const`` and ``use strict`` together

**Comment by fna** [comment]
>/home/travis/build/cfpb/owning-a-home/node_modules/jsdom/lib/jsdom/living/helpers/validate-names.js:2
const xnv = require("xml-name-validator");

It looks like it's part of a node module, we could probably skip the failing tests, or we could contact the author of the module and ask him to update his code. It should be just a sec for @ascott1 @contolini to apply their magic.

**Comment by cfarm** [comment]
The tests are breaking ever since we merged the [expandables PR](https://github.com/cfpb/owning-a-home/pull/352) to master. I'm working on a fix, but this branch can merge since it didn't introduce the bug and the bug is already on build.

File: bower.json [hunk 1]
```diff
@@ -11,8 +11,9 @@
     "cf-forms": "git://github.com/cfpb/cf-forms.git#1.2.0",
     "cf-grid": "git://github.com/cfpb/cf-grid.git#0.9.0",
     "cf-layout": "git://github.com/cfpb/cf-layout.git#0.2.2",
-    "cf-pagination": "git://github.com/cfpb/cf-pagination.git#0.4.1",
+    "cf-pagination": "git://github.com/cfpb/cf-pagination.git#0.4.1",    
     "cf-icons": "git://github.com/cfpb/cf-icons.git#0.4.0",
+    "cf-typography": "git://github.com/cfpb/cf-typography.git#0.7.0",
```
Comment (cfarm) [inline bower.json]: so cool.

**Comment by stephanieosan** [comment]
Excellent. Merge it, @cfarm!

**Pull request merged**